ocaml/typing/jkind_intf.ml

```diff
@@ -253,6 +253,14 @@ module History = struct
 
   type bits64_creation_reason = Primitive of Ident.t
 
+  type top_creation_reason =
```

Comment:
Now that these are reasons for `top`, do any stop being used for `any`? I would expect `Missing_cmi` and `Dummy_jkind` at least to stop being used for `any`.

Reply:
I'd rather not make any changes like this, as I found that filling in `top`s by my judgment doesn't seem to have worked out well in practice, and I sometimes need to move it back to using `any`. Let's do removals like that last.